src/scripts.js

```diff
@@ -1 +1,73 @@
-console.log('Hello world');
+let currentUser;
+let currentRecipe;
+let ingredientList = document.querySelector('.ingredients-view');
+let recipeName = document.querySelector('.recipe-name');
+let recipeView = document.querySelector('.recipe-view');
+let recipeInstructions = document.querySelector('.recipe-instructions');
+let recipeImg = document.getElementById('recipe-img');
+let searchButton = document.querySelector('.search-button');
+let randomRecipeName = document.querySelector('.random-recipe-name');
+let randomRecipeImage = document.getElementById('large-dish-image');
+let userPantryItems = document.querySelector('.pantry-items')
+let userName = document.querySelector('.user-name')
+
+
+window.onload = () => {
+  displayOnPageLoad();
+}
+searchButton.addEventListener('click', () => {displayRecipe(recipe)});
+
+
+// FOR HOME PAGE
+function displayOnPageLoad() {
+    displayUser();
+    displayRandomRecipe();
+    displayPantryItems();
+}
+
+function getRandomUser() {
+  let userIndex = Math.floor(Math.random() * usersData.length);
+  currentUser = new User(usersData[userIndex]);
+}
+
+function getRandomRecipe() {
+  let recipeIndex = Math.floor(Math.random() * currentUser.recipes.recipeBook.length);
+  currentRecipe = currentUser.recipes.recipeBook[recipeIndex];
+}
```

Comment:
We can refactor these into one getRandom function and pass in either recipe or user for the argument.